newsletters/2023-11-08-newsletter.md

```diff
@@ -0,0 +1,70 @@
+* [ZKPs Reshape The Way We Think About Identity](https://hackernoon.com/zkps-reshape-the-way-we-think-about-identity) - article by Technology News. (04.11.2023)
+* [Understanding Zero Knowledge Proofs: Safeguarding Privacy in the Digital Age](https://hackernoon.com/understanding-zero-knowledge-proofs-safeguarding-privacy-in-the-digital-age) - article by Samiran Mondal. (06.11.2023)
+* [Malleable Commitments from Group Actions and Zero-Knowledge Proofs for Circuits based on Isogenies](https://eprint.iacr.org/2023/1710.pdf) - academix text by Mingjie Chen, Yi-Fu Lai, Abel Laval, Laurane Marco and Christophe Petit. (06.11.2023)
+* [zkVM Exposed: The Not-So-Secret Sauce Behind General-Purpose Verifiable Computation](https://medium.com/@mrkylenoome/zkvm-exposed-the-not-so-secret-sauce-behind-general-purpose-verifiable-computation-488f91c5594e) - article by Kyle Noome. (06.11.2023)
+* [A Privacy Revolution in Web3 With Zero-Knowledge Proofs (ZKPs)](https://hackernoon.com/a-privacy-revolution-in-web3-with-zero-knowledge-proofs-zkps) - article by Vladislav Bilay. (07.11.2023)
+* [Zero-Knowledge Proofs: It's Like Magic, but I'll Explain It](https://hackernoon.com/zero-knowledge-proofs-its-like-magic-but-ill-explain-it) - article by Phil Windley. (07.11.2023)
+* [Enhancing Identity Management with Zero-Knowledge Proofs in Blockchain](https://medium.com/@solidity101/enhancing-identity-management-with-zero-knowledge-proofs-in-blockchain-937d53f0b531) - article by Solifity Academy. (07.11.2023)
+* [Zero Knowledge Proofs: What's a zkOracle?](https://hackernoon.com/zero-knowledge-proofs-whats-a-zkoracle) - article by Marlene. (07.11.2023)
+* [Unlocking the Magic of Zero-Knowledge Proofs: Disintegration of Philosophy and Science (Part 1)](https://hackernoon.com/unlocking-the-magic-of-zero-knowledge-proofs-disintegration-of-philosophy-and-science-part-1) - article by Antică Vlad. (07.11.2023)
+* [Public report of Sui's zkLogin audit](https://www.zksecurity.xyz/blog/posts/zklogin/) - report by ZKSecurity. (07.11.2023)
+
+### ZK projects
+* [Scroll.io Community Update: October](https://scroll.io/blog/community-update-october). (01.11.2023)
+* [zkSync release: v0.1.0-alpha.10](https://github.com/matter-labs/era-test-node/releases/tag/v0.1.0-alpha.10). (01.11.2023)
+* [QED switched to Poseidon2](https://twitter.com/QEDProtocol/status/1719663267642270049). (01.11.2023)
+* [Polygon CDK: Your Guide to Sovereign L2 Scaling, From A to ZK](https://polygon.technology/blog/polygon-cdk-your-guide-to-sovereign-l2-scaling-from-a-to-zk). (01.11.2023)
+* [Dusk | Release Cycle Update #28](https://dusk.network/news/release-cycle-update-28). (02.11.2023)
+* [Ruby Protocol Monthly Review — October 2023](https://rubyprotocol.medium.com/ruby-protocol-monthly-review-october-2023-ae8463584716). (02.11.2023)
+* [Aztec urges users of zk.money to withdraw any remaining funds](https://twitter.com/aztecnetwork/status/1720459569489723878). (03.11.2023)
```

Comment:
Leave this link.
In addition add https://medium.com/aztec-protocol/sunsetting-aztec-connect-a786edce5cae to articles and make a top pick.